remo/voting/views.py

```diff
@@ -34,22 +34,19 @@ def list_votings(request):
             poll_groups += ['Review']
         polls = Poll.objects.filter(valid_groups__name__in=poll_groups)
 
-    past_polls_query = polls.filter(end__lt=now())
     current_polls = polls.filter(start__lt=now(), end__gt=now())
     future_polls = polls.filter(start__gt=now())
 
-    past_polls_paginator = Paginator(past_polls_query, settings.ITEMS_PER_PAGE)
-    past_polls_page = request.GET.get('page', 1)
-
-    try:
-        past_polls = past_polls_paginator.page(past_polls_page)
-    except PageNotAnInteger:
-        past_polls = past_polls_paginator.page(1)
-    except EmptyPage:
-        past_polls = past_polls_paginator.page(past_polls_paginator.num_pages)
+    show_all_past = request.GET.get('show_all_past','false')
 
+    if show_all_past == 'true':
+        past_polls = polls.filter(end__lt=now())
+    else :
+        past_polls = polls.filter(end__lt=now())[:10]
```

Comment:
Sorry @Ryuno-Ki , i didn't quite understand what you meant. The previous assignee for this bug stated that after consultation with @akatsoulas , it should show 10 of the latest previous votes, and show all previous votes after clicking on a link. Why is there a need to pass in 10 as a per_page variable?